### PR TITLE
Bump ocp version for CI lane

### PR DESCRIPTION
**Pull request by SchSeba**
this should make the ocp lane more stable

### CODE REVIEW

**Review by zeeke — APPROVED** [review]
LGTM

Maybe after 4.16 GA we can stick to a GA version. WDYT?

File: hack/run-e2e-conformance-virtual-ocp.sh [hunk 1]
```diff
@@ -206,6 +206,24 @@ podman build -t "${SRIOV_NETWORK_CONFIG_DAEMON_IMAGE}" -f "${root}/Dockerfile.sr
 echo "## build webhook image"
 podman build -t "${SRIOV_NETWORK_WEBHOOK_IMAGE}" -f "${root}/Dockerfile.webhook" "${root}"
 
+echo "## wait for the all cluster to be stable"
+MAX_RETRIES=20
+DELAY_SECONDS=10
+retries=0
+until [ $retries -ge $MAX_RETRIES ]; do
+  if [ $(kubectl get clusteroperator --no-headers | awk '{print $3}' | grep True | wc -l) -eq 33 ]; then
```
Comment (adrianchiris) [inline hack/run-e2e-conformance-virtual-ocp.sh:214]: nit: please add comment what it is we are checking here

Comment (SchSeba) [reply hack/run-e2e-conformance-virtual-ocp.sh:215]: done :)

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**